lib/web_ui/test/text_editing_test.dart

```diff
@@ -549,6 +549,41 @@ void main() {
       expect(spy.messages, isEmpty);
     });
 
+    test('negative base and offset values in editing state is handled', () {
```

Comment:
Should "base and offset" be "base offset and selection extent"?

Reply:
done.